src/components/todosList.tsx

```diff
@@ -0,0 +1,26 @@
+import { useContext } from 'react';
+import { Todo } from '../types/Todo';
+import { TodosItem } from './todosItem';
+import { TodosContext } from './todosContext';
+
+interface ListProps {
+  items: Todo[];
+}
+
+export const TodosList: React.FC<ListProps> = ({ items }) => {
+  const { todos } = useContext(TodosContext);
+
+  const isListShown = () => {
+    return todos.length > 0;
+  };
+
+  return (
+    isListShown() === true && (
```

Comment:
```suggestion
  return (
    todos.length > 0 && (
```
we can use it directly

Reply:
or like that
```suggestion
if(!todos.length){
 return null
}

  return (
```